src/app/components/api-key/api-key-registration.test.jsx

```diff
@@ -0,0 +1,225 @@
+import React from 'react';
+import * as angularReactHelper from '../../services/angular-react-helper.jsx';
+import { render, cleanup, screen, waitFor, fireEvent } from '@testing-library/react';
+import { toBeInvalid, toBeValid, toBeEnabled, toBeDisabled, toHaveValue } from '@testing-library/jest-dom';
```

Comment:
I've been trying to alphabetize the imports where I can. I don't know if we can lint that, but I think it'd be helpful for consistency

```suggestion
import { toBeDisabled, toBeEnabled, toBeInvalid, toBeValid, toHaveValue } from '@testing-library/jest-dom';
```